infra/vsts_agent_ubuntu_20_04.tf

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+data "template_file" "vsts-agent-ubuntu_20_04-startup" {
+  template = "${file("${path.module}/vsts_agent_ubuntu_20_04_startup.sh")}"
+
+  vars = {
+    vsts_token   = "${secret_resource.vsts-token.value}"
+    vsts_account = "digitalasset"
+    vsts_pool    = "ubuntu_20_04"
+  }
+}
+
+resource "google_compute_region_instance_group_manager" "vsts-agent-ubuntu_20_04" {
+  provider           = "google-beta"
+  name               = "vsts-agent-ubuntu-20-04"
+  base_instance_name = "vsts-agent-ubuntu-20-04"
+  region             = "us-east1"
+  target_size        = 1
```

Comment:
Once that gets merged I'll be redeploying the tf file with a 10 here (and make a new PR for that, and the git checkout thing).